src/data/faq.json

```diff
@@ -1748,17 +1749,16 @@
                                 "anchor": "international_phone_numbers",
                                 "active": false,
                                 "textblock": [
-                                    "<b>Updated January 18, 2023</b>",
+                                    "<b>Updated February 1, 2023</b>",
                                     "You can reach us by phone from within Germany and from abroad. The national numbers are free of charge, but are only available from within Germany. The international numbers are <b>not free of charge</b> - the cost of the international call is based on the contract with your provider.",
                                     "<b>Technical Hotline</b>",
                                     "The technical hotline will help you with technical questions about the Corona-Warn-App, for instance if you have problems with the exposure logging (only available in German). It is available Monday to Friday from 9 a.m. to 5 p.m. German local time (except on German national holidays):",
                                     "<b>National:</b> 0800 7540001",
                                     "<b>International:</b> +49 30 498 75401",
                                     "<b>TAN Hotline</b>",
-                                    "The TAN hotline will help you to enter the TAN if you have a positive PCR test result and if you have not received a TAN or document with a PCR test QR code. The TAN hotline is available Monday to Sunday from 7 a.m. to 8 p.m. German local time:",
-                                    "<b>National:</b> 0800 7540002",
-                                    "<b>International:</b> +49 30 498 75402",
-                                    "Please note that as of version 3.0 of the Corona-Warn-App, a teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of January 31, 2023.",
+                                    "The operation of the TAN hotline has been discontinued as of January 31, 2023. Previously, it helped you enter the TAN if you had a positive PCR test result and if you did not receive a TAN or a document with a PCR test QR code. You were able to call it from Monday to Sunday from 7 a.m. to 8 p.m. local time in Germany:",

```

Comment:
Same here regarding operating hours.

Reply:
Updated in https://github.com/corona-warn-app/cwa-website/pull/3357/commits/b006857fcb5fdb081ce62f9f90250306044fa7f1, thanks for the suggestion!